getting-started/create-a-mobile-property.md

```diff
@@ -77,24 +79,24 @@ To deploy your configuration to a development environment for testing:
 
    The library builds and is displayed under the **Development** section of the publishing workflow.
 
-7. On the library card, click _..._ to see a drop-down list.
+7. On the library card, click the ellipsis (...) to see a dropdown list.
 8. Select **Submit for Approval** and then **Submit**.
 
-The library of changes are then published to the Development environment and the library is displayed under the **Submitted** section of the publishing workflow.
+The library of changes are then published to the development environment and the library is displayed under the **Submitted** section of the publishing workflow.
 
 {% hint style="info" %}
-Testing can be done using the configuration in the Development environment. The library can later be deployed to the **Staging** and **Production** environments by using the rest of the publishing workflow. For more information, see [Publishing](https://docs.adobe.com/content/help/en/launch/using/reference/publish/overview.html).
+Testing can be done using the configuration in the development environment. The library can later be deployed to the **Staging** and **Production** environments by using the rest of the publishing workflow. For more information, see the documentation on [publishing in Launch](https://docs.adobe.com/content/help/en/launch/using/reference/publish/overview.html).
```

Comment:
@calebk1, Development is a proper name here. We have three environments created by default called "Development", "Staging" and "Production". They can, however, create additional development environments and call them something other than "Development". With that context, not sure if we capitalize "Development" here or not?

Reply:
Ah ok, yup that makes sense. I'll re-capitalize it then.